.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
```

Comment:
It's better not to commit vscode settings changes with your PR's. Adopting a python formatter would have to be a team decision.